include/syslog.h

```diff
@@ -121,6 +121,10 @@
 #define LOG_INFO      6  /* Informational message */
 #define LOG_DEBUG     7  /* Debug-level message */
 
+#define LOG_LOWOUT    8  /* Log to console straightly
```

Comment:
This will still conflict with the POSIX definitions per my earlier comment unfortunately. Maybe you could use a mask in the higher bits? Like (1 << 31)?

This way the priority still can indicate the syslog level and subsystem, but the extra bit is a non-POSIX extension that allows you to log to the console directly for debugging.

I'm still not sure if that's the best way to do it, because there are already existing macros for each subsystem to syslog that won't be able to log to your new sink unless they're modified to add that bit mask. It would only really help you if you're quickly debugging and you need to add in some temporary 'syslog' calls yourself that you remove later. That being said, I don't really have any better ideas. Maybe you could also try asking the mailing list.